src/ServiceControl.Monitoring.Data/LongValueWriter.cs

```diff
@@ -3,7 +3,7 @@
     using System;
     using System.IO;
 
-    static class LongValueWriter
+    public static class LongValueWriter
```

Comment:
If these are going to be versioned, should there be V1, V2, etc. concrete implementations with a factory method to get one of them?